README.md

```diff
@@ -44,3 +44,61 @@ Make sure you have branched off of the latest version of master. Keep commits sh
 There is a pre-commit hook that will check for good code style, and fail commits with an error. Run `npm run lint` to find out more.
 
 Please check for Markdown linting issues if you're updating the `README.md` or any `Markdown`.
+
+### Lighthouse
```

Comment:
Maybe title the section "Performance Metrics & Insights" and simply add a sentence before Local Testing mentioning that we use Lighthouse for metrics and analysis.